packages/eas-cli/src/commandUtils/flagHelpers.ts

```diff
@@ -0,0 +1,41 @@
+import { Flags } from '@oclif/core';
```

Comment:
My thinking was to locate any shared logic having to do with flags here (not necessarily just pagination flags). I can make this change, but in the future, if someone wants to know where to put some logic that has to do with flags, they would no longer consider this file. I think it would be good to keep the name as is (or a similar name)

Reply:
1. I think this is a premature "optimization".
2. Generally, it's always better to have specialized modules instead of a bag for "utils". If you want to go with this approach, it'd be better to make a new directory in `commandUtils` and rename the file to sth with "pagination" string.

Reply:
yeah, i felt like flags was a sufficiently narrow scope for a utility file. I do agree about overly generalized files being a poor practice. it's nbd, I'll rename.